lib/src/Pos/Context/Context.hs

```diff
@@ -61,21 +55,6 @@ import           Pos.Util.UserSecret (HasUserSecret (..), UserSecret)
 -- NodeContext
 ----------------------------------------------------------------------------
 
-data LastKnownHeaderTag
```

Comment:
There is no documentation to move, unfortunately.

Reply:
:pepe_sad: